### PR TITLE
TVAULT-3240

### PR DESCRIPTION
**Pull request by shyam-biradar**

### CODE REVIEW

**Comment by shyam-biradar** [comment]
@abhijeetpatra @amitkumar-a4  Please review.

File: kolla-ansible/ubuntu/docker/trilio-datamover/nova-sudoers [hunk 1]
```diff
@@ -1,2 +1 @@
-nova ALL = (root) NOPASSWD: /home/tvault/.virtenv/bin/privsep-helper *
```
Comment (shyam-biradar) [inline kolla-ansible/ubuntu/docker/trilio-datamover/nova-sudoers]: @amitkumar-a4 @abhijeetpatra : Let me know, if this change is fine. This is optional. But to remove unnecessary privileges, I am removing this from sudoers.

**Comment by amitkumar-a4** [comment]
@shyam-biradar  Have you tested all the snapshot/restore scenario's because in description Nitesh has mentioned that even after having both privsep(system and contego venv) entry in nova sudoers, snapshots were failing.

**Comment by shyam-biradar** [comment]
@amitkumar-a4 : Issue was due to incorrect privsep helper command path in tvault-contego.conf.
Tested everything. What we have not tested is removing entry from sudoers for our privsep helper. For that I asked for your input. We will test that fix too.

**Pull request merged**